deploy/charts/mixer/templates/hpa.yaml

```diff
@@ -0,0 +1,50 @@
+{{- range $index, $unused := .Values.serviceGroups }}
+{{- range $serviceName, $unused := . }}
+{{- if gt $index 0 }}
+---
+{{- end  }}
```

Comment:
nit: should these come after the license header?

Reply:
yes they should, thanks!